core/src/sample_performance_service.rs

```diff
@@ -75,14 +77,18 @@ impl SamplePerformanceService {
                 let highest_slot = bank_forks.highest_slot();
                 drop(bank_forks);
 
-                let perf_sample = PerfSample {
-                    num_slots: highest_slot
-                        .checked_sub(sample_snapshot.num_slots)
-                        .unwrap_or_default(),
-                    num_transactions: bank
-                        .transaction_count()
-                        .checked_sub(sample_snapshot.num_transactions)
-                        .unwrap_or_default(),
+                let num_slots = highest_slot;
+                let num_transactions = bank.transaction_count();
+                // This value will come from `bank.non_vote_transaction_count_since_restart()` in a
+                // separate change.
+                let num_non_vote_transactions = 0u64;
```

Comment:
We can update this now that https://github.com/solana-labs/solana/pull/29383 has landed right?

Reply:
I was not sure where to put this change.
It is currently in the third commit: [`f5c80b4` (#29388)](https://github.com/solana-labs/solana/pull/29388/commits/f5c80b4bb76a3824ff240ec6faaa6276915ddb43#diff-65bb3c028302211b6b87741af7f24b648cc1e4949a3d1ef7c8e6f5ee28744b22)
I can move it here, if you think it makes more sense here.

Reply:
Done.